pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -567,6 +572,7 @@ func (p *provider) Create(machine *v1alpha1.Machine, data *cloudprovidertypes.Pr
 				SourceType:          bootfromvolume.SourceImage,
 				UUID:                image.ID,
 				VolumeSize:          *c.RootDiskSizeGB,
+				VolumeType:          c.RootDiskVolumeType,
```

Comment:
The comment at https://pkg.go.dev/github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/bootfromvolume#BlockDevice is talking about `	// This requires Compute API microversion 2.67 or later.`, which openstack that corresponds to?

Reply:
That's Openstack Stein which covers API microversions up to 2.72.